r-package/grf/R/quantile_forest.R

```diff
@@ -171,6 +185,12 @@ predict.quantile_forest <- function(object,
     stop("Error: Quantiles must be in (0, 1)")
   }
 
+  # If possible, use pre-computed predictions.
+  quantiles.orig <- object[["quantiles.orig"]]
+  if (is.null(newdata) & identical(quantiles, quantiles.orig) & !is.null(object$predictions)) {
```

Comment:
Would `&&` be better than `&` here?

Reply:
I kept it consistent with [this](https://github.com/grf-labs/grf/blob/master/r-package/grf/R/regression_forest.R#L249), but I agree it is not consistent with the style choice in input_utilities.R. I can bring all of them up to date? (it does not matter in these examples as only single-values are compared)

Reply:
In case it's helpful context -- for single values I prefer && and || since they short-circuit.

Reply:
I was planning on sending a follow-up PR making all of these consistent.